### PR TITLE
Add unknown image dimension sample

### LINKED ISSUE #864: New sample: handling images with unknown dimensions
see https://github.com/ampproject/amphtml/issues/1939

### PR DESCRIPTION
**Pull request by sebastianbenz**
Fixes #864

### CODE REVIEW

**Comment by sebastianbenz** [comment]
Here is a preview: https://amp-by-example-staging.appspot.com/advanced/how_to_support_images_with_unknown_dimensions/

File: src/30_Advanced/How_to_support_Images_with_unknown_Dimensions%3F.html [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    <div class="fixed-container m1">
+  <amp-img layout="fill" src="https://unsplash.it/200/300?image=1074"></amp-img>
+    </div>
+ </div>
+
+ <!-- The question is: how can images with unknown dimensions be embeded inside an AMP pages while maintaining their correct aspect ration? 
+
+    ## Object-Fit to the Rescue
+    
+    We can solve this problem by combining AMP's [fill layout](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-layout.md) with the [object-fit]( https://css-tricks.com/almanac/properties/o/object-fit/) CSS property. The `object-fit` property helps us ensure that the image maintains it's aspect ratio while being resized via the `fill` layout.
```
Comment (kul3r4) [inline src/30_Advanced/How_to_support_Images_with_unknown_Dimensions%3F.html]: it's aspect ration -> its aspect ration

File: src/30_Advanced/How_to_support_Images_with_unknown_Dimensions%3F.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  <div class="flex">
+    <div class="fixed-container m1">
+  <amp-img layout="fill" src="https://unsplash.it/300/200?image=1074"></amp-img>
+    </div>
+    <div class="fixed-container m1">
+  <amp-img layout="fill" src="https://unsplash.it/200/300?image=1074"></amp-img>
+    </div>
+ </div>
+
+ <!-- The question is: how can images with unknown dimensions be embeded inside an AMP pages while maintaining their correct aspect ration?
```
Comment (kul3r4) [inline src/30_Advanced/How_to_support_Images_with_unknown_Dimensions%3F.html]: ration -> ratio

Comment (sebastianbenz) [inline src/30_Advanced/How_to_support_Images_with_unknown_Dimensions%3F.html]: done

File: src/30_Advanced/How_to_support_Images_with_unknown_Dimensions%3F.html (hunk 1 above)
Comment (sebastianbenz) [inline src/30_Advanced/How_to_support_Images_with_unknown_Dimensions%3F.html]: done

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

**Pull request merged**